packages/mrmime/mrmime.0.1.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+name:         "mrmime"
+version:      "dev"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/mirage/mrmime"
+bug-reports:  "https://github.com/mirage/mrmime/issues"
+dev-repo:     "git+https://github.com/mirage/mrmime.git"
+doc:          "https://mirage.github.io/mrmime/"
+license:      "MIT"
+synopsis:     "Mr. MIME"
+description:  """Parser and generator of mail in OCaml"""
+
+build: ["dune" "build" "-p" name "-j" jobs]
+run-test: ["dune" "runtest" "-p" name "-j" jobs]
+
+pin-depends: [
+  [ "crowbar.dev" "git+https://github.com/stedolan/crowbar.git" ]
+]
+
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency anymore (see #14266)

Reply:
Oh ok I did not know!